### ISSUE TITLE
User story - Job list

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#32]
### If applicable, add a related Epic.

Epic: #28 

### User Story

As a **user**, I can **view a list of created job posts** so that I can **view open positions**.

### Acceptance Criteria

- There should be a way to click into the "Job list" view and see the listed jobs
- The jobs should be posted by most recent

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Create a job list template
- [x] Populate jobs in cards and order them
- [x] Consider adding pagination

**Comment by stephendawsondev** [comment]
This was completed with PR #83 and #85
